### ISSUE TITLE
Block Backpropagation

### ISSUE THREAD

**Issue by Sowasa** [vlfeat/matconvnet#84]
Hello,
Thank you for this great package, very inspiring!
Now my question: Is it possible to block backpropagation easily when I only want to retrain part of the network form an existing model? 
Say I want to keep layer 1-3 from ImageNet training and then randomly initialize layer 4-7 and only train those, but keep the first 3. Is there an option to do this easily?
Best regards,
Sowasa

**Comment by dteney** [comment]
You can just set the learning rate of those layers to 0. To go faster, you can also modify the eval_simplenn function to stop backpropagating once it reaches the layers that don't need to be updated.
